packages/jspsych/src/index.scss

```diff
@@ -58,11 +58,24 @@
   font-size: 14px;
 }
 
+/* Buttons and Button Groups */
+
+.jspsych-btn-group-flex {
+  display: flex;
+  justify-content: center;
+}
+
+.jspsych-btn-group-grid {
+  display: grid;
+  grid-auto-columns: max-content;
+  min-width: fit-content;
```

Comment:
This doesn't shrink the buttons to the largest button's minimum size. I'd prefer adding
```
  max-width: fit-content;
  margin-right: auto;
  margin-left: auto;
```
here so
![Screenshot B](https://github.com/jspsych/jsPsych/assets/25661029/b5db1f80-09b7-4649-a38a-90de945fcccc)
becomes
![Screenshot B](https://github.com/jspsych/jsPsych/assets/25661029/5a11d314-d9f3-4051-b43b-66f4d0d9fcf4)